### PR TITLE
MUZIMA-697

### PR DESCRIPTION
**Pull request by samuelowino**
- [x] Provided alternative message if no patient is available locally.

- [x] Translated alternative message to French, Portuguese and Swahili

### CODE REVIEW

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -245,6 +245,7 @@
     <string name="header_muzima_category">Paramètres de mUzima</string>
     <string name="header_openmrs_category">Paramètres de OpenMRS</string>
     <string name="header_provider_location_category">Paramètres du fournisseur / emplacement</string>
+    <string name="info_no_client_avaialble_localy">Aucun client disponible localement. Saisissez au moins 3 caractères pour la recherche à distance ou enregistrez de nouveaux clients.</string>
```
Comment (mssavai) [inline app/src/main/res/values-fr/strings.xml:248]: @samuelowino please rename the string resource to _info_no_client_available_locally_

File: app/src/main/res/values-sw/strings.xml [hunk 2]
```diff
@@ -749,7 +749,7 @@
     <string name="warning_relationship_type_missing">" Tafadhali chagua aina ya uhusiano"</string>
     <string name="warning_session_timeout">Muda wa sesheni kuisha: Haikuweza kuweka muda wa kusitisha.
        Tafadhali ingiza thamani kati ya 1 na 500. </string>
-    <string name="info_no_client_avaialble_localy">Hakuna wateja wanaopatikana ndani ya nchi. Ingiza angalau herufi 3 kwa utaftaji wa mbali au sajili wateja wapya.</string>
+    <string name="info_no_client_available_locally">Hakuna wateja wanaopatikana ndani ya nchi. Ingiza angalau herufi 3 kwa utaftaji wa mbali au sajili wateja wapya.</string>
```
Comment (samuelowino) [inline app/src/main/res/values-sw/strings.xml:752]: Might need to change this translation :)

Comment (samuelowino) [reply app/src/main/res/values-sw/strings.xml:752]: Can we go with;
"Hakuna wateja wanaopatikana ndani ya simu. Ingiza angalau herufi 3 kwa utaftaji wa mbali au sajili wateja wapya kwenye simu"

Comment (samuelowino) [reply app/src/main/res/values-sw/strings.xml:752]: @mssavai let me get your thoughts

Comment (mssavai) [reply app/src/main/res/values-sw/strings.xml:752]: A agree

Comment (samuelowino) [reply app/src/main/res/values-sw/strings.xml:752]: Great let get on that 👌

**Pull request merged**